identifiers.md

```diff
@@ -2,12 +2,31 @@
 
 **Content**
 
+* [Introduction to identifiers](#introduction-to-identifiers)
 * [eventID](#eventid)
 * [occurrenceID](#occurrenceid)
 
+### Introduction to identifiers
+
+Using a unique identifier for each event, physical sample, or subsample in your dataset taken at each location and time is highly recommended to ensure sample traceability and data provenance. For OBIS, the two main identifiers of concern are `occurrenceID` and `eventID`. However if you look at [TDWG's Darwin Core reference guide](https://dwc.tdwg.org/terms/), you may see that there are several other terms for identifiers. When creating identifiers for your data, or mapping an existing identifier field to Darwin Core vocabulary, you may hesitate between choosing between `eventID`, `occurrenceID`, `organismID`, `taxonID`, `scientificNameID`, `recordNumber`, `materialEntityID`, `materialSampleID`, and `catalogNumber`. Knowing the difference between these and when to use each may seem confusing at first. Review the table below for a quick comparison of definitions and when to use each DwC identifier term. Note this table is not exhaustive of all possible DwC identifer terms, but includes many that are relevant for occurrence records.
+
+|Identifier Name | Definition | When to Use |
+|----|----|----|
+| `eventID` | An identifier for the set of information associated with a dwc:Event (something that occurs at a place and time). May be a global unique identifier or an identifier specific to the data set. | Used to distinguish between events in your data, *not* associated with an occurrence, e.g. a quadrat sample, an ROV deployment, a trolling event within a cruise |
+| `occurrenceID` | An identifier for the dwc:Occurrence (as opposed to a particular digital record of the dwc:Occurrence). In the absence of a persistent global unique identifier, construct one from a combination of identifiers in the record that will most closely make the dwc:occurrenceID globally unique. | Used to distinguish between occurrence records, i.e. when the presence of a taxon is detected (or not for absence records). If the same individual is detected multiple times, each detection will have unique a `occurrenceID` |
+| `organismID` | An identifier for the dwc:Organism instance (as opposed to a particular digital record of the dwc:Organism). May be a globally unique identifier or an identifier specific to the data set. | Use to identify a *specific* organism, an individual, or a specific group of organisms (e.g. a specific pod of cetaceans, a specific shark) |
+| `taxonID` | An identifier for the set of dwc:Taxon information. May be a global unique identifier or an identifier specific to the data set. | Used to identify a specific taxonomic rank, not commonly used in OBIS because `scientificNameID` is prioritized |
```

Comment:
when to use: I think the wording "Used to identify a specific taxonomic rank" can be improved because this may be misunderstood as the the identifier of dwc:taxonRank. I don't know how to explain this clear enough that people will/can distinguish the difference between taxonID and scientificNameID 😂 To be fair, I am the one who asked a question related to this in dwc-qa https://github.com/tdwg/dwc-qa/issues/203 Sorry that this comment is not very constructive.

Marie from GBIF wrote a very good blog post: https://data-blog.gbif.org/post/some-checklist-terms-explained/ but I don't know how to relate that to Occurrence datasets because OBIS does not harvest checklists.